docs/model-serving/quickstart.md

```diff
@@ -0,0 +1,175 @@
+The library is Apache V2 licensed and available [here](https://github.com/logicalclocks/machine-learning-api). It currently comes with a Python SDK.
+If you want to connect to the Model Serving from outside Hopsworks, see our [integration guides](../integrations/setup.md).
+
+The library is build around metadata-objects, representing entities within Model Serving. You can modify metadata by changing it in the metadata-objects and subsequently persisting it to the Model Serving. In fact, the Model Serving itself is also represented by an object. Furthermore, these objects have methods to operate with the deployments in the Model Serving.
+
+
+### Guide Notebooks
+
+This guide is based on a [series of notebooks](https://github.com/logicalclocks/hops-examples/tree/master/notebooks/ml), which are available in the Deep Learning Demo Tour Project on Hopsworks.
+
+
+### Connection, Project and Model Serving
+
+The first step is to establish a connection with your Hopsworks Model Serving instance and retrieve the object that represents the Model Serving you'll be working with.
+
+By default `connection.get_model_serving()` returns the Model Serving of the project you are working with.
+
+=== "Python"
+
+    ```python
+    import hsml
+
+    # Create a connection
+    connection = hsml.connection()
+
+    # Get the model serving handle for the project's model serving
+    ms = connection.get_model_serving()
+    ```
+
+
+### Integration with the Model Registry
+
+The Model Serving library integrates smoothly with the Model Registry library. Metadata objects such as [Model](../generated/model-registry/model.md) can be used in multiple methods in the Model Serving library to simplify the code and avoid duplication of parameters.
+
+As a result, the easiest way to deploy an existing model is using the `model.deploy()` method available in the model metadata object. For instance, the version with highest accuracy of a trained model can be deployed with the following lines.
+
+=== "Python"
+
+    ```python
+    # Get the model registry handle for the project's model registry
+    mr = connection.get_model_registry()
+
+    # Get the version with highest accuracy
+    model = mr.get_best_model('mnist', 'accuracy', 'max')
+
+    # Create a deployment
+    deployment = model.deploy()
+    ```
+
+To learn more about managing models with the Model Registry see the [Model Registry documentation](../model-registry/quickstart.md).
+
+
+### Deployment
+
+Assuming you have already created a model in the Model Registry, a deployment can now be created to prepare a model artifact for this model and make it accessible for running predictions behind a REST endpoint. A model artifact is a package containing all necessary files for the deployment of a model, including the model file and/or custom scripts for loading the model (a.k.a. predictor script) or transforming the model inputs at inference time (a.k.a. transformer script).
```

Comment:
a.k.a. -> a.k.a. the

Reply:
Actually, i would just drop the 'a.k.a' and write
the predictor script
the transformer script